klippy/extras/display/uc1701.py

```diff
@@ -61,13 +67,14 @@ def init(self):
                      0x2E, # Voltage regulator on
                      0x2F, # Voltage follower on
                      0xF8, # Set booster ratio
-                     0x00, # Booster ratio value (4x)
+                     0x20, # Booster ratio value (4x)
```

Comment:
According to the UC1701 datasheet valid values for booster ratio are 0x00 - 0x03.  If a display needs it changed from 0x00 (which I don't know why it would as its a NOP) then it should probably be a configuration option.